KromerPlotLib.py

```diff
@@ -0,0 +1,338 @@
+import pandas as pd
+import numpy as np
+import astropy.units as u
+import astropy.constants as uconst
+import pyne
+import matplotlib.cm as cm
+import matplotlib
+
+
+
+try:
+    import astropy.modeling.blackbody as abb
+except ImportError:  # for astropy version < 2.0
+    import astropy.analytic_functions as abb
+import matplotlib.pyplot as plt
+
+
+
+def constructKromerPacketFrame(sim):
+    """
+    Constructs Pandas DataFrames from a TARDIS
+    Simulation object containing logged packet
+    properties used for Kromer analysis.
+
+    Parameters
+    ----------
+    sim : tardis.simulation
+        TARDIS simulation object to analyze
+
+    Returns
+    -------
+    kromer_packet_df : pd.DataFrame
+        DataFrame containing tardis packet logged properties
+
+    """
+    packet_nus = sim.runner.virt_packet_nus * u.Hz
+    packet_nus = packet_nus.to('Hz', u.spectral())
+    last_line_interaction_in_nu = sim.runner.virt_packet_last_interaction_in_nu
+
+    packet_lambdas = packet_nus.to('angstrom', u.spectral())
+    packet_energies = sim.runner.virt_packet_energies * u.erg
+
+    last_interaction_type = sim.runner.virt_packet_last_interaction_type
+    last_line_interaction_out_id = sim.runner.virt_packet_last_line_interaction_out_id
+    last_line_interaction_in_id = sim.runner.virt_packet_last_line_interaction_in_id
+
+    kromer_packet_df = pd.DataFrame(
+        np.column_stack(
+            (packet_nus,
+             packet_lambdas,
+             packet_energies,
+             last_interaction_type,
+             last_line_interaction_out_id,
+             last_line_interaction_in_id,
+             last_line_interaction_in_nu)),
+
+        columns=['nus',
+                 'lambdas',
+                 'energies',
+                 'last_interaction_type',
+                 'last_line_interaction_out_id',
+                 'last_line_interaction_in_id',
+                 'last_line_interaction_in_nu']
```

Comment:
```suggestion
        {
            'nus': packet_nus,
            'lambdas': packet_lambdas,
            'energies': packet_energies,
            'last_interaction_type': last_interaction_type,
            'last_line_interaction_out_id': last_line_interaction_out_id,
            'last_line_interaction_in_id': last_line_interaction_in_id,
            'last_line_interaction_in_nu': last_line_interaction_in_nu
        }
```
I think this can possibly be a better way.